src/annotations/components/annotationForm/AnnotationStartTimeInput.tsx

```diff
@@ -17,10 +18,25 @@ interface Props {
 }
 
 export const AnnotationStartTimeInput: FC<Props> = (props: Props) => {
-  const validationMessage = props.startTime ? '' : 'This field is required'
+  // initial input value is the startTime passed
+  const [inputValue, setInputValue] = useState<string>(
```

Comment:
`inputValue` is a decent name, but not superb. It describe how it's used fine, which makes the code easier to follow, but if this was named more after what it is, it'll be even easier to follow along and remember the purpose of this code. What I mean is, this is the start time input for annotations, is there a name you can think of more descriptive than `inputValue` that might represent what the `inputValue` is here?

Reply:
Yes absolutely! I see your point and this makes sense. I'm thinking `startTimeValue`